packages/installer/optifine.ts

```diff
@@ -81,9 +81,9 @@ export function installByInstallerTask(installer: string, minecraft: MinecraftLo
         let zip = await open(installer);
         context.update(10, 100);
 
-        let entry = zip.entries["net/optifine/Config.class"];
+        let entry = zip.entries["net/optifine/BlockDir.class"];
```

Comment:
You are right. This should be 
```ts
zip.entries["net/optifine/Config.class"]
            ?? zip.entries["Config.class"];
```

On old version, the config class is on the root.

Reply:
Sorry if this is not how you are supposed to do pull requests btw, this is my first pull request ever :(